packages/flutter/lib/src/widgets/toggleable.dart

```diff
@@ -320,7 +323,7 @@ mixin ToggleableStateMixin<S extends StatefulWidget> on TickerProviderStateMixin
       enabled: isInteractive,
       onShowFocusHighlight: _handleFocusHighlightChanged,
       onShowHoverHighlight: _handleHoverChanged,
-      mouseCursor: mouseCursor.resolve(states),
+      mouseCursor: mouseCursor?.resolve(states) ?? SystemMouseCursors.basic,
```

Comment:
shouldn't this default to `.click`?. In either case, please document the default

Reply:
Will document the default. Cupertino toggleables default to `.basic` (by not allowing a `mouseCursor` to be passed in at all) and Material toggleables (previously) required a `WidgetStateProperty` that defined what `mouseCursor`s to display depending on the `MaterialState` of the widget.